libcontainer/rootfs_linux.go

```diff
@@ -400,12 +400,25 @@ func mountToRootfs(m *configs.Mount, c *mountConfig) error {
 			return err
 		}
 		// Selinux kernels do not support labeling of /proc or /sys
+		if m.IsMove() && *c.fd >= 0 {
```

Comment:
Nit: unlike in C, you don't have to dereference a pointer to a struct when accessing its members.

IOW `s/*c.fd/c.fd/`